docs/design/free_credit_offer_typesDesignDoc.md

```diff
@@ -56,37 +56,39 @@ The credit is automatically applied to the account and will have a max limit tha
 **Referrer Award Credit**:
   - This is the credit awarded to the Referrer once the Invitee has created a *paid* account and has successfully paid their second invoice. This credit can have a different expiration date than the awarded Invitee Credit and the Default Free Credit.
 
-[Offer Types Table](https://docs.google.com/spreadsheets/d/1I3Do-HMNkpUpJAsebtl1NXw6-PlkoVaFKCP9e-TTgHA/edit?zx=aqgnyh2ltfad#gid=0&range=A3:E13)
+[Offer Types Table](https://docs.google.com/spreadsheets/d/1I3Do-HMNkpUpJAsebtl1NXw6-PlkoVaFKCP9e-TTgHA/edit?usp=sharing)
 
 ## Design
 
 ### Database
 
 **offer  table**
 ```sql
-    Id - bytea
+    Id - int
     Name -  text
     Description - text  
     Credits - integer
     redeemable_cap - integer
     Num_redeemed - integer
     Created_at - timestamp
-    offer_expiration - int
-    award_credit_expiration - int
-    Invitee_credit_expiration - int
+    offer_duration - interval
```

Comment:
Looks like you'll have to go back to int after all :)

I'd recommend specifying what the expected units are in the field name, though. Maybe something like "offer_duration_seconds" or "offer_duration_days" or whatever.